### ISSUE TITLE
Short freezes in combat

### ISSUE THREAD

**Issue by alex-north** [jordonwow/bigdebuffs#602]
Ever since 10.0.5, I have short freezes = screen freezes up for like 0.5-1sec while in combat. This is especially true in party and raid groups, M+ and Arena. I tried turning off all addons which solved the problem. Then I tried every addon one by one and it seems like BigDebuffs is causing it. The Blizzard raid frames also go crazy when someone is joining or leaving a raid group during elite WQs.

**Comment by jonomurphy** [comment]
Experiencing this too

**Comment by neuvo123** [comment]
confirm very troublesome

**Comment by RostumAhady** [comment]
Yep, in a raid setting with BigDebuffs on my frame rate drops massively. Turned of all addons and only had BigDebuffs and it was definitely the cause of it.

**Comment by EnigmaPatrick** [comment]
Seems like it's causing mine too, unable to see any errors or reasons why this is.

**Comment by aza547** [comment]
same here

**Comment by citna** [comment]
I was able to get around the raid setting massive FPS drops by using the very first DF release (11.3) with the latest Mainline spell list file placed into it. All I can notice from trying different versions in LFR is that as soon as the compact raid or party frame taint/lua error, your FPS immediately drops. The latest versions cause that to happen immediately on you entering combat and the 11.3 version does not.
I don't think it's a real solution because if someone leaves and joins while in combat and triggers an error, your fps is cut until a /reload. I haven't been able to test in shuffles, multiple arena matches, or battlegrounds yet. I only tested in a couple skirmishes with no problems.

**Comment by StreeJezuz** [comment]
@citna Just testing the 11.3 version and its doing the same thing when entering combat and not even in group.

Sadly its a entering combat issue with this addon after patch 10.5, hopefully the author can fix it as its very annoying in arena. I was streaming on twitch last night, doing Shuffles and have a link to exact issue effecting entering combat after rounds of groups changes. Normally happens from round 3 or 5, anyone interested take a look as i had to /reload mid combat to fix the issue.

Street_Jesus_6
https://www.twitch.tv/videos/1719940580?t=02h26m46s

I hope this helps the Author

**Comment by jordonwow** [comment]
Thanks for the info, trying to get this fixed quickly

**Comment by citna** [comment]
"Fixed an issue that caused performance decreases when using certain addons" was in today's hotfix notes for Retail but possibly won't be implemented or confirmed as a fix until after server restarts.
